docs/data/date-pickers/time-picker/time-picker.md

```diff
@@ -43,46 +39,65 @@ The component can be uncontrolled or controlled
 
 {{"demo": "TimePickerValue.js"}}
 
-## Responsiveness
+## Available components
 
-The Time picker Component is designed and optimized for the device it runs on.
+The component is available in four variants:
 
-- The `MobileNextTimePicker` component works best for touch devices and small screens.
-- The `DesktopNextTimePicker` component works best for mouse devices and large screens.
+- The `DesktopNextTimePicker` component which works best for mouse devices and large screens.
+  It renders the views inside a popover and allows editing values directly inside the field.
 
-By default, the `NextTimePicker` component renders the desktop version if the media query [`@media (pointer: fine)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) matches.
-This can be customized with the `desktopModeMediaQuery` prop.
+- The `MobileNextTimePicker` component which works best for touch devices and small screens.
+  It renders the view inside a modal and does not allow editing values directly inside the field.
 
-There are certain caveats when testing pickers, please refer to [this section](/x/react-date-pickers/getting-started/#testing-caveats) for more information.
+- The `NextTimePicker` component which renders `DesktopNextTimePicker` or `MobileNextTimePicker` depending on the device it runs on.
+
+- The `StaticTimePicker` component which renders without the popover/modal and field.
 
 {{"demo": "ResponsiveTimePickers.js"}}
 
+By default, the `NextTimePicker` component renders the desktop version if the media query [`@media (pointer: fine)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) matches.
+This can be customized with the `desktopModeMediaQuery` prop.
+
+:::warning
+Responsive components can suffer some inconsistencies between testing environments if media query is not supported.
+Please refer to [this section](/x/react-date-pickers/getting-started/#testing-caveats) for solutions.
+:::
+
 ## Static mode
 
-It is also possible to render any Time Picker without the modal/popover and text field.
+It is also possible to render the Time Picker component without the popover/modal and field.
```

Comment:
Did you decide to keep this section separate for Time Picker because of the fact that `desktop` picker doesn't have a view? 🤔

Reply:
No I just missed it, my bad :grimacing:

Reply:
I believe the demo is now still missing the Static variant. 🙈 
And in other pages (date and date-time) we've changed component labels to have `variant` in them. 🤔